### PR TITLE
Remove mention of ZIPs from spec

### PR DESCRIPTION
**Pull request by awoods**
Resolves: https://github.com/OCFL/spec/issues/188

### CODE REVIEW

File: draft/spec/index.html [hunk 1]
```diff
@@ -768,24 +767,9 @@ <h2>Storage Hierarchies</h2>
           <li>Storage hierarchies MUST be terminated by OCFL Object Roots</li>
           <li>Storage hierarchies within the same OCFL Storage Root SHOULD use just one layout pattern</li>
           <li>Storage hierarchies within the same OCFL Storage Root SHOULD consistently use either a directory
-            hierarchy of OCFL Objects, top-level OCFL Objects, or OCFL Object ZIPs
+            hierarchy of OCFL Objects or top-level OCFL Objects
```
Comment (rosy1280) [inline draft/spec/index.html]: missing a period.

Comment (zimeon) [reply draft/spec/index.html]: there aren't periods on the other bullets... all or none!

Comment (awoods) [reply draft/spec/index.html]: I am not a fan of periods in lists.

Comment (awoods) [reply draft/spec/index.html]: comedy

Comment (ahankinson) [reply draft/spec/index.html]: 😁

**Comment by ahankinson** [comment]
Will treat @rosy1280 's comment as a '-1' for now and will not merge.

File: draft/spec/index.html (hunk 1 above)
Comment (rosy1280) [reply draft/spec/index.html]: I could have sworn I saw periods in the list.  oh well.  FINE!

**Pull request merged**